roles/splunk_common/tasks/add_splunk_license.yml

```diff
@@ -2,42 +2,28 @@
 - name: Download Splunk license
   get_url:
     url: "{{ splunk.license_uri }}"
-    dest: /tmp/splunk.lic
+    dest: "{{ splunk.license_download_dest }}"
     timeout: 20
   when:
-    - splunk.license_uri is defined
-    - splunk.license_uri is search("^http.*")
```

Comment:
You might need this condition added back in? If `splunk.license_uri` is null, the get_url would fail?

Reply:
The get_url is only used when the user specifies a http link with --splunk-local-license. If it fails, the lookup further down will simply skip the download location and revert to the default (nfr) license.